app/views/lessons/_lesson_buttons.html.erb

```diff
@@ -10,6 +10,28 @@
   <%= link_to 'Next Lesson',
       course_lesson_path(course, lesson.next_lesson, ref: 'lnav'),
       title: "Move on to '#{lesson.next_lesson.title}'",
-      class: "button #{next_lesson_button_state(lesson)} lesson-button-group__item lesson-button"
+      class: "button #{next_lesson_button_state(lesson)} lesson-button-group__item lesson-button",
+      id: "next-lesson-button"
   %>
 <% end %>
+ 
+<script>
+  // Fire an event whenever a user marks a lesson completed on the individual lesson page
+  // else marks a lesson uncompleted on the individual lesson page
+  $('#submission-button').on('click', function(e) {
```

Comment:
I think we can use attribute selectors for the title attribute to find the complete and incomplete buttons instead of adding an id to the buttons:

something like
`$('[title="Mark lesson incomplete"]').on('click', function(e) {`
...
`$('[title="Mark lesson complete"]').on('click', function(e) {`